x-pack/test/functional/services/ml/common_table_service.ts

```diff
@@ -101,6 +102,42 @@ export function MlTableServiceProvider({ getPageObject, getService }: FtrProvide
         `Filtered table should have ${expectedRowCount} row(s) for filter '${queryString}' (got ${rows.length} matching  items)`
       );
     }
+
+    public async assertTableSorting(
+      columnName: string,
+      columnIndex: number,
+      expectedDirection: 'desc' | 'asc'
+    ) {
+      const actualDirection = await this.getCurrentSorting();
+      expect(actualDirection?.direction).to.eql(expectedDirection);
+      expect(actualDirection?.columnName).to.eql(columnName);
+    }
+
+    public async getCurrentSorting(): Promise<
+      { columnName: string; direction: string } | undefined
+    > {
+      const table = await testSubjects.find(`~${this.tableTestSubj}`);
+      const headers = await table.findAllByClassName('euiTableHeaderCell');
+      for (const header of headers) {
+        const ariaSort = await header.getAttribute('aria-sort');
+        if (ariaSort !== 'none') {
+          // TODO Add support for column names with _
```

Comment:
Is this TODO meant to be addressed for this PR?

Reply:
Implemented in https://github.com/elastic/kibana/pull/153462/commits/ceb2bc5c3366097bacfba3283cc5a063b06a08bb